pkgs/applications/altcoins/particl/particl-core.nix

```diff
@@ -29,12 +29,17 @@ stdenv.mkDerivation rec {
     unixtools.hexdump
   ];
 
-  configureFlags = [ "--with-boost-libdir=${boost.out}/lib" ];
+  configureFlags = [
+    "--disable-bench"
+    "--enable-tests=no"
```

Comment:
Did we run the tests before?

Reply:
I think this flag only to build tests, and it was never used (not sure about it).
I trust @tecnovert (particl-core developer) to run all tests before publishing release.

Reply:
I added `doCheck` that run tests, also running tests required python3 which was added: 233dad81ebc164520fb763cbb47a16c88655888d